lib/Core/SymbolicError.h

```diff
@@ -63,9 +63,13 @@ class SymbolicError {
   /// \brief The path probability
   double pathProbability;
 
+  /// \brief The path length
+  int branchCount;
+
 public:
   SymbolicError() : pathProbability(-1.0) {
     errorState = ref<ErrorState>(new ErrorState());
+    branchCount = 0;
```

Comment:
@Himeshi You can do this in C++ style by adding `, branchCount(0)` after `pathProbability(-1.0)`. Same can be said with `errorState`, actually. But please keep in mind, when initialized, using the list, the variables need to be in the same order they are defined in the class.